### DISCUSSION TITLE
Push Paketo Buildpacks to CNB registry

### DISCUSSION THREAD

**Discussion by kvedurmu** [paketo-buildpacks/feedback#1] [Ideas]
We should look into publishing all meta/implementation Paketo buildpacks to the CNB registry, now that it's been implemented as per [this RFC](https://github.com/buildpacks/registry-index). 

This would involve running `pack register-buildpack` or using leveraging existing [github actions](https://github.com/buildpacks/github-actions#add) everytime a new buildpack is published to GCR/Dockerhub. Ultimately, publishing all buildpacks to the CNB registry will make it easier for users to "discover" buildpacks that exist.

**Comment by kvedurmu** [comment]
cc @sophiewigmore this just duplicates https://github.com/paketo-buildpacks/github-config/issues/204. 

I'm wondering if the github discussions feature gives us a good way to surface project-wide requests such as this.

**Comment by sophiewigmore** [comment]
This idea was implemented in: https://github.com/paketo-buildpacks/github-config/pull/222!
Any buildpack that has a release cut now will be pushed to the CNB registry
